### ISSUE TITLE
val_loss frozen at  0.2000

### ISSUE THREAD

**Issue by nzinfo** [foamliu/FaceNet#3]
When I started training process, the val_loss fixed.
Epoch 00001: val_loss improved from inf to 0.20000, saving model to models/model.01-0.2000.hdf5
Epoch 2/15
 211/1539 [===>..........................] - ETA: 48:51 - loss: 0.1927
1539/1539 [==============================] - 3427s 2s/step - loss: 0.1902 - val_loss: 0.2000

Epoch 00002: val_loss did not improve from 0.20000
Epoch 3/15
1539/1539 [==============================] - 3425s 2s/step - loss: 0.1872 - val_loss: 0.2000

Epoch 00003: val_loss did not improve from 0.20000
Epoch 4/15
1539/1539 [==============================] - 3430s 2s/step - loss: 0.1844 - val_loss: 0.2000

Epoch 00004: val_loss did not improve from 0.20000
Epoch 5/15
1539/1539 [==============================] - 3434s 2s/step - loss: 0.1820 - val_loss: 0.2000

Epoch 00005: val_loss did not improve from 0.20000
Epoch 6/15
1539/1539 [==============================] - 3415s 2s/step - loss: 0.1794 - val_loss: 0.2000

Epoch 00006: val_loss did not improve from 0.20000

Epoch 00006: ReduceLROnPlateau reducing learning rate to 4.999999873689376e-06.
Epoch 7/15
1539/1539 [==============================] - 3411s 2s/step - loss: 0.1777 - val_loss: 0.2000

Epoch 00007: val_loss did not improve from 0.20000
Epoch 8/15
1539/1539 [==============================] - 3397s 2s/step - loss: 0.1775 - val_loss: 0.2000

Epoch 00008: val_loss did not improve from 0.20000
Epoch 9/15
1539/1539 [==============================] - 3410s 2s/step - loss: 0.1761 - val_loss: 0.2000

Epoch 00009: val_loss did not improve from 0.20000
Epoch 10/15
1539/1539 [==============================] - 3410s 2s/step - loss: 0.1754 - val_loss: 0.2000

Have no idea about it.

**Comment by KaiHuangMO** [comment]
it can be easily solved by changing verbose to  zero
